app/views/ucas_contacts/show.html.erb

```diff
@@ -31,110 +31,11 @@
-              <%= @provider.web_link_contact[:name] %>
-              <%= @provider.web_link_contact[:email] %>
-              <%= @provider.web_link_contact[:telephone] %>
-            <% else %>
-              Information unknown
-            <% end %>
-          </p>
-          <p class="govuk-body-s">
-            This person will receive notices about UCAS systems and planned downtime
-          </p>
-        </dd>
-        <dd class="govuk-summary-list__actions"></dd>
-      </div>
-
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          Finance contact
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <p class="govuk-body" data-qa="provider__finance_contact">
-            <% if @provider.finance_contact.present? %>
-              <%= @provider.finance_contact[:name] %>
-              <%= @provider.finance_contact[:email] %>
-              <%= @provider.finance_contact[:telephone] %>
-            <% else %>
-              Information unknown
-            <% end %>
-          </p>
-          <p class="govuk-body-s">
-            This person will receive invoices from UCAS for payment of fees
-          </p>
-        </dd>
-        <dd class="govuk-summary-list__actions"></dd>
-      </div>
-
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          Fraud contact
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <p class="govuk-body" data-qa="provider__fraud_contact">
-            <% if @provider.fraud_contact.present? %>
-              <%= @provider.fraud_contact[:name] %>
-              <%= @provider.fraud_contact[:email] %>
-              <%= @provider.fraud_contact[:telephone] %>
-            <% else %>
-              Information unknown
-            <% end %>
-          </p>
-          <p class="govuk-body-s">
-            This person will receive alerts about fraudulent activity (for example when a plagiarised personal statement is detected)
-          </p>
-        </dd>
-        <dd class="govuk-summary-list__actions"></dd>
-      </div>
-
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          UCAS Administrator
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <p class="govuk-body" data-qa="provider__admin_contact">
-            <% if @provider.admin_contact.present? %>
-              <%= @provider.admin_contact[:name] %>
-              <%= @provider.admin_contact[:email] %>
-              <%= @provider.admin_contact[:telephone] %>
-            <% else %>
-              Information unknown
-            <% end %>
-          </p>
-          <p class="govuk-body-s">
-            This person is responsible for setting up and maintaining UCAS user accounts.
-          </p>
-        </dd>
-        <dd class="govuk-summary-list__actions"></dd>
-      </div>
+      <% if @provider.contacts.present?  %>
```

Comment:
That looks wrong


https://s121d02-12-ptt-as.azurewebsites.net/organisations/T92/ucas-contacts